### DISCUSSION TITLE
Getting error by using lang=lesss

### DISCUSSION THREAD

**Discussion by HenriqueAurelio** [less/less.js#3687] [Q&A] [answered]
![Imagem_25-01-2022_as_21 32](https://user-images.githubusercontent.com/31359644/151098916-22df7955-ec43-4f22-b9db-b945c7a25700.jpg)
![Imagem_25-01-2022_as_21 30](https://user-images.githubusercontent.com/31359644/151098918-7a63195c-7ac3-495b-b2cd-96135e064607.jpg)

Someone can help me? I searched and found https://github.com/nativescript-vue/nativescript-vue/issues/480 this but didnt work.

**Comment by iChenLei** [comment, marked as answer]
where is the error msg ?

**Reply by HenriqueAurelio** [reply]
Sorry, getting this error https://cdn.discordapp.com/attachments/859981490926256179/935741312022511627/Imagem_26-01-2022_as_00.41.jpg

**Reply by iChenLei** [reply]
@HenriqueAurelio  https://github.com/webpack-contrib/less-loader/blob/master/CHANGELOG.md#600-2020-04-24
> move less-specific options to the lessOptions option, please look at [README](https://github.com/webpack-contrib/less-loader#lessoptions)
